packages/lesswrong/server/resolvers/tagHistoryFeed.ts

```diff
@@ -31,19 +34,22 @@ defineFeedResolver<Date>({
     const {currentUser} = context;
     
     const tagRaw = await Tags.findOne({_id: tagId});
+    if (!tagRaw) throw new Error("Tag not found");
     const tag = await accessFilterSingle(currentUser, Tags, tagRaw, context);
     if (!tag) throw new Error("Tag not found");
     
     type SortKeyType = Date
     
-    const lenses = await MultiDocuments.find({
+    const lensesAndSummaries = await MultiDocuments.find({
       parentDocumentId: tagId,
-      fieldName: "description",
     }).fetch();
+    const lenses = lensesAndSummaries.filter(md => md.fieldName==="description");
     const lensIds = (historyOptions.lensId && historyOptions.lensId !== "all")
       ? [historyOptions.lensId]
       : lenses.map(lens => lens._id);
-      
+    const summaries = lensesAndSummaries.filter(md => md.fieldName==="summary");
```

Comment:
We need to also fetch the summaries for the lenses, not just the top-level comment